### DISCUSSION TITLE
new shutter commads??

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#22507] [General]
Hello .. 
 I just built new version ot the dev. release ([Tasmota 14.3.0.7 (TasmoCompiler-esp82662M) and I am glad to se the new GUI layout for the shuttercontrol .. VERY well done. nice and thank you. Very nice is also to the command `Color20  ` to control color of Button when Off; also very nice and useful.
Now my question is:
Is it possible to set the name of the shutter?? in the UI it shows "`Shutter 1`" .. so I think their will be (in the future) command to set the name.

Regards
Mark-Muc
![shutter](https://github.com/user-attachments/assets/5463afcb-c019-4bf4-ac4a-88f9e11c2f77)

**Comment by arendst** [comment]
The name can be changed with webbutton comand of the right button. So if you use relay1 and relay2 use webbutton1

**Reply by Mark-Muc** [reply]
oh thanks . that was fast and IT WORKS!!
Regards Mark-Muc

BTW: Why did you change in the UI  the up / down buttons??  
If i look at my buttons (switches) the up button is left und the down is right!. as it was before.
![Rolloschalterjpg](https://github.com/user-attachments/assets/c388ebe7-e1df-4f9d-8436-f92b6899da10)
